com.unity.ml-agents/Runtime/Actuators/VectorActuator.cs

```diff
@@ -55,6 +73,11 @@ public void OnActionReceived(ActionBuffers actionBuffers)
             m_ActionReceiver.OnActionReceived(ActionBuffers);
         }
 
+        public void Heuristic(in ActionBuffers actionBuffersOut)
+        {
+            m_HeuristicProvider.Heuristic(actionBuffersOut);
```

Comment:
```suggestion
            m_HeuristicProvider?.Heuristic(actionBuffersOut);
```
since `actionReceiver as IHeuristicProvider` could result in null?

Reply:
huh, had that in there before. not sure why I removed it.